MANIFEST.in

```diff
@@ -1 +1,7 @@
-include README.rst LICENSE *.txt
\ No newline at end of file
+include README.rst LICENSE requirements*.txt
+include runtests.py
```

Comment:
I guess this is the file which is impacting the `coverage`, we should exclude it from being considered.

Reply:
I've removed it, and since Makefile uses it, removed that also.